front/app/utils/paginationUtils.ts

```diff
@@ -1,13 +1,18 @@
-import { parse } from 'qs';
-import { isNil } from 'lodash-es';
+import { parse } from 'query-string';
+import Url from 'url-parse';
```

Comment:
We are normally using `qs` for parsing urls - why do we change it here? Also I'm not sure why we are introducing a new dependency - `url-parse` instead of just using what we already have? Is there some value there I'm missing?

Ideally, we should make  this work with our current setup and without new dependencies.